### PR TITLE
external/webclient : fix an error when entering a long domain name.

### PR DESCRIPTION
**Pull request by JeonginKim**
When using wget, error occurs when domain name is long.

### CODE REVIEW

**Comment by seinfra** [comment]
Code Rule Check OK.

File: external/webclient/webclient.c [hunk 1]
```diff
@@ -106,8 +106,8 @@
 
 #define CONFIG_WEBCLIENT_MAXHTTPLINE 200
 #define CONFIG_WEBCLIENT_MAXMIMESIZE 32
-#define CONFIG_WEBCLIENT_MAXHOSTNAME 40
-#define CONFIG_WEBCLIENT_MAXFILENAME 100
+#define CONFIG_WEBCLIENT_MAXHOSTNAME 128
```
Comment (pillip8282) [inline external/webclient/webclient.c]: how about to define it at Kconfig. usually CONFIG_* is set in Kconfig.

**Comment by seinfra** [comment]
Code Rule Check OK.

**Comment by seinfra** [comment]
Code Rule Check OK.

**Comment by seinfra** [comment]
Code Rule Check OK.

**Pull request merged**